src/client.js

```diff
@@ -1,7 +1,15 @@
+
+// DISABLE CONSOLE LOG IN PRODUCTION
+if (window.location.hostname !== 'localhost'){
+	console.log = function() {}
```

Comment:
this is not a safe way to disable `console.log`
babel can be instructed to strip `console.log` calls during build process.

Reply:
I added the fix

Reply:
It's a better approach but it misses other log methods such as `console.warn` or `console.error`.
I was thinking of adding this plugin to babel and have it take care of all of it, or configure it the way it suits us:

https://babeljs.io/docs/en/babel-plugin-transform-remove-console/